src/views/blocks/event-datetime.php

```diff
@@ -80,34 +81,36 @@
 			<span class="tribe-events-schedule__all-day"><?php echo __( 'All day', 'the-events-calendar' ); ?></span>
 		<?php endif; ?>
 
-		<?php if ( ! $is_all_day || ! $is_same_day ) : ?>
-			<span class="tribe-events-schedule__separator tribe-events-schedule__separator--time">
-				<?php echo $separator_time; ?>
-			</span>
-		<?php endif; ?>
-
-		<?php if ( ! $is_same_day ) : ?>
-			<span class="tribe-events-schedule__date tribe-events-schedule__date--end">
-				<?php echo $formatted_end_date; ?>
-			</span>
+		<?php if ( ! $is_same_start_end ) : ?>
+			<?php if ( ! $is_all_day || ! $is_same_day ) : ?>
+				<span class="tribe-events-schedule__separator tribe-events-schedule__separator--time">
+					<?php echo $separator_time; ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$separator_time'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$separator_time'.

<sub>hash: 10fbd97efdabbac037705ce58622e6d7</sub>